thanks.py

```diff
@@ -20,28 +20,30 @@
     'Thank you!! :D',
     'Thanks so much!',
     'Thanks!',
-    'Thank you! I appreciate it!'
+    'Thank you! I appreciate it!',
+    'Thank you! <3'
 ]
 
 # Define our error-based sleep fallback timer
 sleep_time = 1
 
-
 # Define our ridiculous "birthday" query
 def build_birthday_fql(min_date, max_date, limit = 500):
     birthday_fql = ("SELECT post_id, actor_id, target_id, created_time, message, comments, like_info "
                     "FROM stream "
                     "WHERE source_id = me() "
-                        "AND filter_key = 'others' "
                         "AND created_time > {} "
                         "AND created_time < {} "
-                        "AND actor_id != me() "
```

Comment:
It caused an error in Graph API, when I removed them, the query worked perfectly.